haproxystats/process.py

```diff
@@ -610,6 +611,7 @@ def process_servers(self, data_frame, filter_backend):
         cnt_metrics = 1
         # A filter for rows with stats for servers
         is_server = data_frame['type'] == 2
+        got_traffic = data_frame['lbtot'] > 0
```

Comment:
Same here,  a comment to explain the reason we do this

Reply:
Done as well.